cclibs/blocks/promotion/promotion.js

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2021 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+// export default async function (block, name, document) {
+export default async function init(el) {
+  const locale = 'en-US';
+  const promotionName = el.getAttribute('data-promotion');
+  const response = await window.fetch(`${locale === 'en-US' ? '' : `/${locale}`}/promotions/hub/${promotionName}.plain.html`);
```

Comment:
your locale is always en-US
you probably want it to be different depending on the user's set locale in the config object

Reply:
@fullcolorcoder We were working on moving fedpub CC content from fedpub to CC sharepoint https://jira.corp.adobe.com/browse/MWPW-127801. 
The [promotions.js](https://github.com/adobecom/fedpub/blob/main/hub/blocks/promotion/promotion.js) file has hardcoded the locale value to 'en-US' `const locale = 'en-US';` 
In [initial fedpub](https://github.com/adobe/fedpub) repo this was `const { locale } = window.fedPub;` 
Could you please confirm if this is an expected change and if promitions is always supposed to have en-US content to be pulled or if this needs to be changed during CC migration to pull locale specific content?

Reply:
@aishwaryamathuria currently there are only active "promotions" on en-us content on the original franklin fedpub, you can see what's currently active in the metadata.xlsx. I think it's safe to put as en-us for now and extended later if requested based on what's being used today. However, to get the locale you could pull in the info from utils.

Reply:
@vhargrave As discussed, I have remove the check as en-US is default locale.